src/engine/controllers/buildController.ts

```diff
@@ -67,21 +66,21 @@ export class BuildController {
    */
 
   private static async packageSources(context: Context): Promise<Readable> {
-    const excludedDirectories = [
-      context.config.buildDistFolder,
-      context.config.packageFolder,
-      context.config.metaFolder,
+    const excludedDirectories = ['.git', '.idea'];
+
+    const excludedRoots = [
       context.config.buildRootFolder,
+      context.config.buildDistFolder,
       context.config.modulesFolder,
-      '.git',
-      '.idea',
+      context.config.metaFolder,
+      context.config.packageFolder,
     ];
 
-    const ignoreFilter = fs.existsSync(IGNORE_FILE_PATH)
-      ? ignore().add(fs.readFileSync(IGNORE_FILE_PATH).toString())
-      : {
-          ignores: () => false,
-        };
+    const ignoreFilter = ignore().add(excludedRoots.map(item => '/' + item));
```

Comment:
Why do we have to add '/' symbol ?

Reply:
From docs for ingore:
'/*.js' should only match 'a.js', but not 'abc/a.js'.

Reply:
can we also check OS?

`os.platform() === win32` =>
`map.(el => replase('/' with '\\') )`

Reply:
or maybe it better to rewrite it at config file?